project/REVIEWING.md

```diff
@@ -0,0 +1,170 @@
+remove this label and add the next one. If documentation is absent but expected, maintainers should
+ask for documentation and move to status `status/3-docs-review` for docs maintainer to follow.
+
+Possible transitions from this state:
+
+ * Close
+ * `status/1-design-review`: raises design concerns
+ * `status/3-docs-review`: general case
+ * `status/4-ready-to-merge`: trivial change not impacting documentation
+
+## Docs review - `status/3-docs-review`
+
+Maintainers are expected to review the documentation in its bigger context, ensuring consistency,
+completeness, validity, and breadth of coverage across all extent and new documentation.
+
+They should ask for any editorial change that makes the documentation more consistent and easier to
+understand.
+
+Changes and additions to docs must be reviewed and approved (LGTM'd) by a minimum of two docs
+sub-project maintainers. If the docs change originates with a docs maintainer, only one additional
+LGTM is required (since we assume a docs maintainer approves of their own PR).
+
+Once documentation is approved (see below), a maintainer should make sure to remove this label and
+add the next one.
+
+Possible transitions from this state:
+
+ * Close
+ * `status/1-design-review`: raises design concerns
+ * `status/2-code-review`: requires more code changes
+ * `status/4-ready-to-merge`: general case
+
+## Merge - `status/4-merge`
+
+Maintainers are expected to merge this pull request as soon as possible.  They can ask for a
+rebase, or carry the pull request themselves.  These should be the easy PRs to merge.
+
+Possible transitions from this state:
+
+ * Merge: general case
+ * Close: carry PR
+
+When merging a pull request, the maintainer should consider applying one or multiple impact labels
+to ease future classification (see below).
+
+# Escalation process
+
+Despite the previously described reviewing process, some PR might not show any progress for various
+reasons:
+
+ - No strong opinion for or against the proposed patch
+ - Debates about the proper way to solve the problem at hand
+ - Lack of consensus
+ - ...
+
+All these will eventually lead to stalling PR, where no apparent progress is made accross several
+weeks, or even months.
+
+Maintainers should use their best judgement and apply the `status/needs-attention` label. It must
+be used sparingly, as each PR with such label will be discussed by a group of maintainers during a
+review session. The goal of that session is to agree on one of the following outcomes for the PR:
+
+ * Close, explaining the rationale for not pursuing further
+ * Continue, either by pushing the PR further in the workflow, or by deciding to carry the patch
+   (ideally, a maintainer should be immediately assigned to make sure that the PR keeps continued
+   attention)
+ * Escalate to Solomon by formulating a few specific questions on which his answers will allow
+   maintainers to decide.
+
+# Labels
```

Comment:
I would move this section to the top of the doc since describing what labels are and what they're used for **after** we go thru all of this stuff about what each one is seems backwards.